base_no_translation/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=========================
+Disable model translation
+=========================
+
+Provides means to globally disable translation of field contents for
+all fields on a per Model base.
+Overrides 'translate=True' of all fields of a model (including inherited
```

Comment:
I'm nitpicking here: could you please change these quotes ("'") to backquotes ("`"). These are markers in ReST for `interpreted text` which is generally used for code excerpts. 

Same would need to be done in the rest of the file.